co-simulation/bundle/src/assembly/resources/scenarios/Tiergarten/infrastructure/infrastructure_config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "updateInterval": 1000,
```

Comment:
Last question what is update interval representing and what are its units. Sometimes it's value is 1000 and sometimes its 100. Is that intentional?